docs/extensions.adoc

```diff
@@ -462,6 +462,28 @@ annotated object. To prevent cleanup exceptions from being reported, override th
 def ignoreMyExceptions
 ----
 
+=== TempDir
+
+Generate temp directory for test, and clean them after test.
```

Comment:
You shouldn't mix singular and plural, either `directory` and ´it`, or `directories` and `them`.
But as one annotation will create one directory, I would use singular.

Also, does the extension clean it, or delete it? I think deletion would make more sense.

Also two optional parameters could be handy. One to define the parent directory for the temporary folder and one whether to actually delete it or whether to keep it after execution.
It could be helpful to keep the direcotry to investigate the files after execution. The latter could be helpful as closure, so that it can be determined dynamically, for example using an environment variable.
Maybe with our `PreconditionContext` that we use for `@Requires` and similar as parameter, so that environment variables can be accessed nicely.

Reply:
Thanks, give me one day to apply your advice.

Reply:
Sorry, did accidentally send the first comment standalone instead as complete review. :-)